org.alloytools.alloy.dash/src/main/java/ca/uwaterloo/watform/rapidDash/DashExprToPython.java

```diff
@@ -70,35 +85,41 @@ private String genExpr(Expr node, int size) {
         // TODO: currently, the second parameter is redundant
 
         // check type, there are two types of expr
-        if (node instanceof ExprUnary) {
+        if (node instanceof  ExprList) {
```

Comment:
New stuff, expression list. When we see this, it means this expression has many independent expressions (probably linked with operation "AND").